products/appengine/terraform.yaml

```diff
@@ -51,6 +55,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
       threadsafe: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+      instanceClass: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true    
```

Comment:
Why is this set to `ignore_read`?

Reply:
InstanceClass is not returned during the read, causing the test to fail.

Reply:
@tysen @rileykarson Kindly let me know if there anything I need to do on this. I have tested it locally.

Reply:
@tysen @rileykarson Will appreciate if you can review this asap. There are three PR's which will help us in getting the GAE application use case fully implemented.
#2338 #2269 and #2331 
Thanks.